request.py

```diff
@@ -58,33 +59,36 @@ def from_dict(cls, data: dict):
             thread_id=data["thread_id"],
             title=data["title"],
             end_time=data["end_time"],
-            role = data["role"]
+            role = data["role"],
+            # num_users = data["num_users"],
         )
         instance.bot_message_id = data["bot_message_id"]
         instance.yes_votes = data["yes_votes"]
         instance.no_votes = data["no_votes"]
+        # instance.num_users = data["num_users"]
```

Comment:
We do however still need to be able to save/load it from the state file, so uncommenting this line should be correct.

Reply:
Ah, this. The reason why I commented it out was because it would sometimes throw an exception when I booted up the bot on my test server and there was a `requests_state.json` already generated; it would say that one of the app.py methods (I forgot which one) for some reason wasn't expecting that instance.

...but since the bot should in theory run continuously on astrod's raspberry pi, I don't think its anything to worry about.

Reply:
TLDR yeah feel free to uncomment 👍